### PR TITLE
initial commit adding musicos stats

### PR DESCRIPTION
**Pull request by reimertz**

### CODE REVIEW

File: src/css/main.css [hunk 1]
```diff
@@ -27,3 +27,35 @@
...
+  font-size: 16px;
+  position: relative;
+  max-width: 650px;
+}
+
+.stats div {
+  width: 50%;
+  max-width: 50%;
+  padding: 15px 10px;
+}
```
Comment (il3ven) [inline src/css/main.css]:
```suggestion
.stats {
  margin-top: 50px;
  display: grid;
  grid-template-columns: minmax(50px, 1fr) minmax(50px, 1fr);
  border: 2px solid var(--green);
  color: white;
  box-shadow: 5px 5px 0 white;
  font-size: 16px;
  max-width: 650px;
}

.stats div {
  padding: 15px 10px;
}
```

File: src/layouts/home.njk [hunk 2]
```diff
@@ -7,33 +7,43 @@ layout: base
...
-
-</section>
-
-<a class="all-posts-link" href="{{ '/blog/' | url }}">View All Posts</a>
\ No newline at end of file
+<section class="stats-wrapper">
+  <section class="stats">
+    <div class="b">INDEXED TRACKS</div>
+    <div class="inverted b">{{ musicos.trackCount }}</div>
+    <br/>
```
Comment (il3ven) [inline src/layouts/home.njk]: not needed (at least when using css grid)

```suggestion
```

**Review by il3ven — APPROVED** [review]
LGTM! There is a minor CSS issue causing the table to show black borders on some cells when viewed in mobile.

I propose a solution using CSS grid. I don't have push permissions hence added it as a suggestion but I have tested it locally.

#### Video before and then after CSS grid

https://user-images.githubusercontent.com/4337699/221977599-0fff512e-7310-47f9-8ad7-43e9b66721f3.mov

File: src/layouts/home.njk (hunk 2 above)
Comment (djfnd) [reply src/layouts/home.njk]: The CSS grid solution looks good to me, I noticed that the table broke when on mobile also

**Pull request merged**